clouddriver-kubernetes-v2/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/CustomKubernetesHandlerFactory.java

```diff
@@ -88,7 +89,7 @@ public SpinnakerKind spinnakerKind() {
 
     @Override
     public Manifest.Status status(KubernetesManifest manifest) {
-      return new Manifest.Status();
+      return Status.defaultStatus();
```

Comment:
How would this change effect users who are extending Clouddriver w/ custom CRD handlers? Will these changes break those implementations still using `Manifest.Status`? If so, we should figure out how to alert users to these new changes.

Reply:
The change on this particular line won't affect users at all, as it changes only the implementation of this `status()` function but leaves its behavior identical---`new Manifest.Status()` and `Status.defaultStatus()` are identical.

But the general question is a good one; I've replied in general as non-line-specific PR comment.